.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+-   repo: https://github.com/ambv/black
+    rev: 20.8b1
+    hooks:
+    - id: black
+      language_version: python3
```

Comment:
I have relaxed the `language_version` from `python3.7` to `python3`. Let me know if there is any problem, and I will merge this PR if it's fine.

Reply:
@tczhao

Reply:
@xuyxu yes, works fine